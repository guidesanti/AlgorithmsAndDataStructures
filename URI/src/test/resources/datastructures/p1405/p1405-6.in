98 901
MOV R2,1
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
MOV R3,R2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
IFEQ R2,R3
RET R2
ENDIF
RET R2
98 0
MOV R2,1
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
RET R2
98 2
MOV R2,1
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
RET R2
98 100
MOV R2,1
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
RET R2
98 960
MOV R2,1
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
RET R2
98 999
MOV R2,1
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
MOV R1,R0
DIV R1,R2
ADD R2,R1
DIV R2,2
RET R2
4 0
MUL R0,9
DIV R0,5
ADD R0,32
RET R0
4 0
IFEQ R9,32
RET 1000
ENDIF
RET 1
4 100
MUL R0,9
DIV R0,5
ADD R0,32
RET R0
4 26
MUL R0,9
DIV R0,5
ADD R0,32
RET R0
8 0
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MUL R9,2
RET R9
8 1
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MUL R9,2
RET R9
8 9
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MUL R9,2
RET R9
8 10
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MUL R9,2
RET R9
1 10
RET R0
1 10
RET 450
4 100
MOV R1,R0
ADD R1,1
CALL R1
RET R9
4 999
MOV R1,R0
ADD R1,1
CALL R1
RET R9
9 6
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R2,R9
MUL R2,R0
RET R2
2 123
CALL R0
RET R0
12 6
IFEQ R0,10
CALL R0
ENDIF
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R2,R9
MUL R2,R0
RET R2
12 999
IFEQ R0,10
CALL R0
ENDIF
IFEQ R0,0
RET 1
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R2,R9
MUL R2,R0
RET R2
12 16
IFLE R0,1
RET R0
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R3,R9
MOV R2,R0
SUB R2,2
CALL R2
ADD R3,R9
RET R3
12 256
IFLE R0,1
RET R0
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R3,R9
MOV R2,R0
SUB R2,2
CALL R2
ADD R3,R9
RET R3
12 999
IFLE R0,1
RET R0
ENDIF
MOV R1,R0
SUB R1,1
CALL R1
MOV R3,R9
MOV R2,R0
SUB R2,2
CALL R2
ADD R3,R9
RET R3
19 0
MOV R1,R0
ADD R1,3
MOV R2,R1
MUL R2,6
MOV R3,R2
SUB R3,5
MOV R4,R3
DIV R4,2
MOV R5,R4
MOD R5,4
MOV R6,R5
MUL R6,123
MOV R7,R6
MOV R8,246
ADD R8,R7
MOV R9,R8
ADD R9,507
MOV R0,R9
RET R0
19 0
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 10
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 11
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 20
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 100
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 99
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 50
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 49
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
19 25
IFEQ R0,0
RET 1
ENDIF
IFL R0,11
RET 2
ENDIF
IFLE R0,20
RET 3
ENDIF
IFG R0,99
RET 4
ENDIF
IFGE R0,50
RET 5
ENDIF
IFNEQ R0,25
RET 6
ENDIF
RET 7
5 999
MOV R7,R0
ADD R7,1
SUB R7,1
ADD R9,R7
RET R9
4 0
MOV R4,123
DIV R4,R9
CALL R4
RET R9
5 1
IFNEQ R0,0
CALL 0
CALL 0
ENDIF
RET 3
4 10
IFEQ 0,0
RET 1
ENDIF
RET 0
10 0
IFEQ 999,0
RET 1
ENDIF
IFNEQ 0,0
RET 2
ENDIF
IFG 10,100
RET 3
ENDIF
RET 0
5 10
MOD R0,2
IFNEQ R0,1
RET 1
ENDIF
RET 0
5 657
MOD R0,2
IFNEQ R0,1
RET 1
ENDIF
RET 0
0 0
