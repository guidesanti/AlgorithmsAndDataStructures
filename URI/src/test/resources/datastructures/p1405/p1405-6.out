30
0
1
10
31
0
32
1
212
78
1
2
512
24
10
450
*
*
720
*
720
*
987
867
626
999
1
2
3
3
4
5
5
6
7
999
*
3
1
0
1
0
